projects/storefrontlib/src/lib/store-finder/components/store-finder-store-description/store-finder-store-description.component.spec.ts

```diff
@@ -62,11 +66,7 @@ describe('StoreFinderStoreDescriptionComponent', () => {
 
   function configureTestBed() {
```

Comment:
Please make it consistent with other tests.

Reply:
I refactored this unit test.